guides/ipynb/keras_cv/generate_images_with_stable_diffusion.ipynb

```diff
@@ -37,29 +37,36 @@
     "these powerful performance boosts, and explore the performance benefits\n",
     "that they offer.\n",
     "\n",
+    "**Note:** To run this guide on the `torch` backend, please set `jit_compile=False`\n",
+    "everywhere. XLA compilation for StableDiffusion does not currently work with\n",
```

Comment:
It seems like the AI community typically uses two words, "Stable Diffusion". Let's follow that convention, unless we're talking about a class or other code.